### PR TITLE
update make and updating learning lab

### PR DESCRIPTION
**Pull request by cokieffebah**
# PR Details
update make and emphasizing not to modify pipeline configuration section in first libraries lab

## Description

added two make targets
preview: to make the entire docs using the local changes to sdp-docs and using the other cloned repositories
local-preview: only makes the local sdp-docs changes 

changed earning-labs/1/jte-the-basics/3-first-libraries.html to clarify to *not* update the Pipeline Configuration section when updating the Library Sources configuration

## Types of Changes

- [ ] Fixing spelling errors 
- [ ] Adding a new Learning Lab 
- [x] Updating content to improve clarity 
- [x] This Pull Request does not contain changes to the static HTML in the ``docs`` directory

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,5 +16,9 @@ imagePush:
 	@docker build . -t docker.pkg.github.com/boozallen/sdp-docs/builder
 	@docker push docker.pkg.github.com/boozallen/sdp-docs/builder
 
+local-preview: clean ## builds the antora documentation 	
+	@docker run -v ~/.git-credentials:/root/.git-credentials -v $(shell pwd):/antora:Z --rm -t docker.pkg.github.com/boozallen/sdp-docs/builder generate --stacktrace --generator booz-allen-site-generator --to-dir $(BUILDDIR) $(ANTORABUNDLE) antora-playbook-local-only.yml
```
Comment (steven-terrana) [inline Makefile:20]: this should be updated to reference the `$(LOCAL_PLAYBOOK)` variable

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Minimal makefile to build Antora documentation
 BUILDDIR = docs
 PLAYBOOK = antora-playbook.yml 
+LOCAL_PLAYBOOK = antora-playbook-local.yml
```
Comment (steven-terrana) [inline Makefile:4]: This filename differs from that referenced in `make preview`?

**Review by steven-terrana — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (cokieffebah) [reply Makefile:4]: there are now 3 targets,  docs: makes docs using clone of the repo, 'preview': makes sdp-docs locally, clones the other repos, 'local-preview': which only makes the sdp-docs locally, so I don't have to keep dealing with the zscaler repo clone failure

Comment (cokieffebah) [reply Makefile:4]: added and pushed the 'antora-playbook-local-only.yml'  file

**Review by steven-terrana — APPROVED** [review]

**Pull request merged**